app/src/main/res/xml/preferences.xml

```diff
@@ -61,6 +61,16 @@
             android:entryValues="@array/pref_entryvalues_theme_select"
             android:persistent="true"
             />
```

Comment:
Should the theme selection preference be disabled when aerial imagery is selected?

Reply:
It still affects UI - for example menu of quest question and of settings menu itself.